upup/pkg/fi/clientset_castore.go

```diff
@@ -0,0 +1,630 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fi
+
+import (
+	"bytes"
+	crypto_rand "crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"fmt"
+	"github.com/golang/glog"
+	"golang.org/x/crypto/ssh"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kops/pkg/apis/kops"
+	kopsinternalversion "k8s.io/kops/pkg/client/clientset_generated/clientset/typed/kops/internalversion"
+	"k8s.io/kops/pkg/pki"
+	"k8s.io/kops/util/pkg/vfs"
+	"math/big"
+	"sync"
+	"time"
```

Comment:
Applied `goimports`.  It would change a lot of things to bulk apply it now, but you're right - we should certainly apply it where we can.

(It also isn't perfect, in that it seems to keep existing blocks - but it's better than nothing)